### PR TITLE
[TO REVIEW] Search

### PR DESCRIPTION
**Pull request by N4COM**
Added possibility to use search API
https://docs.2600hz.com/dev/applications/crossbar/doc/search/
However "search" is not a realy Entity, so it should be changed.

### CODE REVIEW

File: lib/hz2600/Kazoo/Api/Entity/Search.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+namespace Kazoo\Api\Entity;
+
+/*
+TODO: Add support for accounts/{account_id}/search
+*/
+
+class Search extends AbstractEntity
+{
+    public function search($type, $value, $view)
```
Comment (swysor) [inline lib/hz2600/Kazoo/Api/Entity/Search.php]: Can you please put opening curly brace on the same line as the function definition to follow the coding style of the project? 
EG: 
function name() {
}

File: lib/hz2600/Kazoo/Api/Entity/Search.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	    return $this -> search("account", $value, $view);
+    }
+    
+    public function getAccountByRealm($realm)
+    {
+	    $result = $this -> searchAccounts($realm, "realm");
+	    
+	    foreach ($result as $account)
+	    {
+		    if ($account -> realm == $realm)
```
Comment (swysor) [inline lib/hz2600/Kazoo/Api/Entity/Search.php]: Please remove spaces between object and method EG: $object->method to comply with style of this project.

File: lib/hz2600/Kazoo/Api/Entity/Search.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Kazoo\Api\Entity;
```
Comment (swysor) [inline lib/hz2600/Kazoo/Api/Entity/Search.php]: I believe we should implement this as a collection

Comment (N4COM) [reply lib/hz2600/Kazoo/Api/Entity/Search.php]: I believe too but I have not tested it as collection so it should be tested before to request pull.

File: lib/hz2600/Kazoo/Api/Entity/Search.php (hunk 1 above)
Comment (N4COM) [reply lib/hz2600/Kazoo/Api/Entity/Search.php]: Done

File: lib/hz2600/Kazoo/Api/Entity/Search.php (hunk 2 above)
Comment (N4COM) [reply lib/hz2600/Kazoo/Api/Entity/Search.php]: Done

File: lib/hz2600/Kazoo/Api/Entity/Search.php (hunk 3 above)
Comment (swysor) [reply lib/hz2600/Kazoo/Api/Entity/Search.php]: Ok, I will close this out for now then. Just open a new pull when you have a version with search implemented as a collection and I will merge it. 

Thank you!

**Pull request closed without merging**